db/migrate/20190624151225_create_outgoings.rb

```diff
@@ -0,0 +1,11 @@
+class CreateOutgoings < ActiveRecord::Migration[5.2]
+  def change
+    create_table :outgoings, id: :uuid do |t|
+      t.string :type_of_outgoing
```

Comment:
`:outgoing_type` ?

Reply:
Yes - that's better. But [the ticket](https://dsdmoj.atlassian.net/browse/AP-734) has "type_of_outgoing" in the JSON input and I think the model name should match that. I'll check with Stephen and see if we can change it.

Reply:
Stephen has confirmed the name can change.

Reply:
I've change the name in the next migration

Reply:
An alternative would be to:
- doing a rollback
- changing the name of this enum and adding the assessment reference
- running the migration again

I think that's acceptable because this is a branch and not deployed to staging or prod. Even if it's deployed to UAT, I would delete the deployment before redeploying

Reply:
It would avoid adding an extra migration to correct a previous migration which is not merged.

Reply:
I don't like modifying migrations across commits, but you're right - it's the merge that's the more important cutoff.

Reply:
You can modify it within the same commit. Create a new commit and then move it around and merge it with the previous commit with an interactive rebase.

Reply:
I've squashed the whole PR.

Reply:
That works too 😄